adwords_client/client.py

```diff
@@ -310,56 +309,44 @@ def _get_service_from_object_type(self, internal_operation):
             return object_type_service_mapper.get(internal_operation['object_type'])
         except KeyError:
             logger.debug('There is no custom service class for this object_type: %s', str(internal_operation['object_type']))
+            raise
 
     def _sync_operations(self):
         previous_client_id = None
         previous_service_name = None
         operation_builder = OperationsBuilder()
         results = []
-        adwords_operations = []
+        adw_ops = []
```

Comment:
Esse var vc pode deixar com o nome grande. Sugeri reduzir o nome somente dentro dos `[op for op in adwords_operations if ...]`

Reduzir o nome dela aqui dificulta a leitura.

A heurística é de que: quanto maior o escopo de uso de um nome, mais vale explicar bem ele. Dentro de um `for` ou `[i for i in ...]` o escopo é bem pequeno, tipo 1-5 linhas. Mas esse aqui vai ser usado por mais de 20 linhas!